### ISSUE TITLE
Package renaming

### ISSUE THREAD

**Issue by tobiasKaminsky** [nextcloud/Android-SingleSignOn#19]
As this library is all about nextcloud, I would like to get rid of owncloud naming.
Is it for you okay, @David-Development, that we use "com.nextcloud.android.sso.*" for all packages?

**Comment by David-Development** [comment]
Yes of course. I thought about it yesterday but it requires a lot of refactoring and renaming :D (especially the aidl interfaces can be very annoying). Feel free to change it! :)

**Comment by David-Development** [comment]
Thank you! :) Looks good from my end. Do you want to merge it or should I merge it?

**Comment by tobiasKaminsky** [comment]
I would say, if you open up a PR and I approve it, then it can get merged (and vice vera).
